app/views/posts/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Create a New Post</h1>
+
+<%= form_for [@user, @post], url: user_posts_path(@user) do |f| %>
+  <div class="field">
+    <%= f.label :text %>
+    <%= f.text_area :text %>
```

Comment:
- [ ] Let's have the title as well because it is necessary. The posts table has a title as an attribute and the controller permits the text field and title as parameters. Without the title field for a post, the table will save a null value which is not good. We need two parameters for a post in this form.